Bender.yml

```diff
@@ -32,6 +32,7 @@ dependencies:
   serial_link:              { git: "https://github.com/pulp-platform/serial_link.git",            version: 1.1.1  }
   tagger:                   { git: "https://github.com/pulp-platform/transaction-tagger.git",     rev: b288376    }
   unbent:                   { git: "https://github.com/pulp-platform/unbent.git",                 version: 0.1.6  }
+  ethernet:                 { git: "git@github.com:pulp-platform/pulp-ethernet.git",              rev: "e3cf735"  }
```

Comment:
Use HTTPS URLs.

Reply:
Also, if you are not using a specific version, please add a comment at the end of the line where you specify to which branch you are pointing at.